types/react-datepicker/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for react-datepicker 3.0
+// Type definitions for react-datepicker 3.1
```

Comment:
It looks the version of the library is still 3.0. So no need to bump it here.

Reply:
I changed it because the [version currently available on npm](https://www.npmjs.com/package/react-datepicker) is 3.1.3. Should I change it back and if so, does it still work under the latest version of react-datepicker?

Reply:
Ahh, that's okay. I just checked the repository and the latest release there is 3.0.

Reply:
Okay thanks!